worlds/AutoWorld.py

```diff
@@ -207,6 +210,9 @@ class Visibility(Enum):
     """The world will be visible/enabled"""
     hidden = enum.auto()
     """The world will be enabled, but hidden"""
+    warning = enum.auto()
```

Comment:
I had been considering the possibility of making this into a bit flag, so that a world could be hidden from webhost/etc (hidden), tests (warning), both (fully disabled), or neither (fully enabled). I think that would probably help with some of the code cleanliness throughout but it's unclear if having "visible on the webhost" and "tests disabled" should ever be an acceptable state

Reply:
If generation is allowed, I think it should be on webhost. Might be something to discuss though.

Reply:
Wait, I was thinking disabled is `user_hidden | test_hidden` in this hypothetical bitflag setup but that is not actually the case. So this is probably fine down to naming.